cmd/tool/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/virtual-vgo/vvgo/pkg/discord"
+	"github.com/virtual-vgo/vvgo/pkg/login"
+	"github.com/virtual-vgo/vvgo/pkg/redis"
+	"github.com/virtual-vgo/vvgo/pkg/sheets"
+	"log"
+	"net/http"
+)
+
+const ApplicationId = "700963768787795998"
+const GuildId = "690626216637497425"
+const RegistrationEndpoint = "https://discord.com/api/v8/applications/" + ApplicationId + "/guilds/" + GuildId + "/commands"
+
+// https://discord.com/developers/docs/interactions/slash-commands#create-guild-application-command
+type CreateApplicationCommand struct {
+	Name        string                             `json:"name"`
+	Description string                             `json:"description"`
+	Options     []discord.ApplicationCommandOption `json:"options,omitempty"`
+}
+
+var beepCommand = CreateApplicationCommand{
```

Comment:
would be good to have these in the same file as their handlers. But that can happen later.

Reply:
Maybe even make a class for DiscordCommand that does the handler and register stuff